tsl/src/remote/row_by_row_fetcher.c

```diff
@@ -116,15 +116,18 @@ row_by_row_fetcher_send_fetch_request(DataFetcher *df)
 		fetcher->state.open = true;
 
 		PGresult *res = PQgetResult(remote_connection_get_pg_conn(fetcher->state.conn));
-		if (!res)
+		if (res == NULL)
```

Comment:
Nothing you need to change, but I usually try to leave these as they are. Any reasonably experience programmers can easily read both manners of null checking.

Reply:
Yep, I just copy-pasted the proper error handling from another place.